settings/templates/settings/personal/security.php

```diff
@@ -66,15 +66,29 @@
 	   href="<?php p(link_to_docs('user-2fa')); ?>"></a>
 	<p class="settings-hint"><?php p($l->t('Use a second factor besides your password to increase security for your account.'));?></p>
 	<ul>
-	<?php foreach ($_['twoFactorProviderData']['providers'] as $data) { ?>
+	<?php foreach ($_['twoFactorProviderData']['providers']  as $data) { ?>
```

Comment:
```suggestion
	<?php foreach ($_['twoFactorProviderData']['providers'] as $data) { ?>
```

Seems to be an extra space which is not needed?